plugins/module_utils/network/eos/config/ospfv3/ospfv3.py

```diff
@@ -147,6 +146,26 @@ def _compare(self, want, have):
 
     def _global_compare(self, want, have):
         for name, entry in iteritems(want):
+            if name == "timers":
+                if entry.get("throttle"):
+                    self._module.warn(
+                        "'timers throttle' command is deprecated by 'timers lsa / timers spf'"
+                    )
+                    entry.pop("throttle")
+                if entry.get("lsa") and not isinstance(entry["lsa"], dict):
```

Comment:
This should probably have another `if isinstance(entry["lsa"], int):` for the modification and error otherwise, just in case someone gets confused and puts a list in here or something. We can safely error if it's not an int because we've never accepted anything else in the past. Something like
```suggestion
                if entry.get("lsa") and not isinstance(entry["lsa"], dict):
                    if not isinstance(entry["lsa"], int):
                        # It doesn't match the new format or the old format, fail here
                        self._module.fail_json(msg="The lsa key takes a dictionary of arguments. Please consult the documentation for more details")
```